makefile

```diff
@@ -516,7 +516,9 @@ ifdef OS_is_win
 $(WORKDIR.lib)/$(core_y:%.dll=%_dll.lib): $(WORKDIR.lib)/$(core_y)
 endif
 $(CORE.tmpdir_y)/$(core_y:%.$y=%_link.txt): $(CORE.objs_y) $(if $(OS_is_win),$(CORE.tmpdir_y)/dll.res,) | $(CORE.tmpdir_y)/. ; $(WRITE.PREREQS)
-$(WORKDIR.lib)/$(core_y):                   $(daaldep.ipp) $(daaldep.vml) $(daaldep.mkl) $(CORE.tmpdir_y)/$(core_y:%.$y=%_link.txt) $(if $(PLAT_is_win32e),$(CORE.srcdir)/export_win32e.def); $(LINK.DYNAMIC) ; $(LINK.DYNAMIC.POST)
+$(WORKDIR.lib)/$(core_y):                   $(daaldep.ipp) $(daaldep.vml) $(daaldep.mkl) \
+                                            $(CORE.tmpdir_y)/$(core_y:%.$y=%_link.txt) \
+                                            $(if $(OS_is_win),$(THR.srcdir)/export.def) ; $(LINK.DYNAMIC) ; $(LINK.DYNAMIC.POST)
```

Comment:
Does `$(if $(OS_is_win),$(THR.srcdir)/export.def)` act the same as `$(if $(PLAT_is_win32e),$(CORE.srcdir)/export_win32e.def)`?

Reply:
No.

There is no reason to have `export_win32e.def`, because symbols enlisted in that file are not required to be exported any more. It's something that we would have removed a couple of years ago.

`export.def` is needed to export MKL symbols and be able to link `onedal.dll` against `daal_core_dll.lib` .